Shield/Shield.csproj

```diff
@@ -18,10 +18,11 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <PackageCertificateKeyFile>Shield_StoreKey.pfx</PackageCertificateKeyFile>
-    <ApplicationInsightsResourceId>/subscriptions/3028579c-d25b-4ab8-87fc-455abc9bb7fb/resourcegroups/AppInsights-MakerProjects/providers/microsoft.insights/components/Windows Virtual Shields for Arduino - Public</ApplicationInsightsResourceId>
+    <ApplicationInsightsResourceId>/subscriptions/3028579c-d25b-4ab8-87fc-455abc9bb7fb/resourcegroups/AppInsights-MakerProjects/providers/microsoft.insights/components/Windows Virtual Shields for Arduino - Internal</ApplicationInsightsResourceId>
```

Comment:
it is worthy now?

On Fri, Aug 28, 2015 at 11:00 AM, Lou Amadio <notifications@github.com>
wrote:

> In Shield/Shield.csproj
> <https://github.com/ms-iot/virtual-shields-universal/pull/21#discussion_r38226632>
> :
>
> > @@ -18,10 +18,11 @@
> >      <FileAlignment>512</FileAlignment>
> >      <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
> >      <PackageCertificateKeyFile>Shield_StoreKey.pfx</PackageCertificateKeyFile>
> > -    <ApplicationInsightsResourceId>/subscriptions/3028579c-d25b-4ab8-87fc-455abc9bb7fb/resourcegroups/AppInsights-MakerProjects/providers/microsoft.insights/components/Windows Virtual Shields for Arduino - Public</ApplicationInsightsResourceId>
> > +    <ApplicationInsightsResourceId>/subscriptions/3028579c-d25b-4ab8-87fc-455abc9bb7fb/resourcegroups/AppInsights-MakerProjects/providers/microsoft.insights/components/Windows Virtual Shields for Arduino - Internal</ApplicationInsightsResourceId>
>
> This should have the public file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ms-iot/virtual-shields-universal/pull/21/files#r38226632>
> .
>